python/lsst/ts/ataos/model.py

```diff
@@ -1,5 +1,10 @@
 
 import numpy as np
+import logging
```

Comment:
Removing these logging setup here. If you want to add a log facility to the model consider adding `log` as an attribute to the class with default `None`. Then, 

```
        if log is None:
            self.log = logging.getLogger(type(self).__name__)
        else:
            self.log = log.getChild(type(self).__name__)
```

Reply:
Sorry, don't quite understand this.
In the boolean evaluation in the if statement and the contents of the else statement, `log` has already been defined somehow so it I suppose I don't understand where/how it gets defined?

Reply:
So, add log as an argument to the model class

```
class Model:
    """ A model class to that handles the ATAOS application level code. This
    class implements the corrections for each of the axis on the AT telescope,
    e.g. M1 and M2 pressure and hexapod correction.

    The AOS corrections are simple polynomes where x is the gravity vector,
    cos(theta), where theta is the azimuth distance or 90.-altitude.

    """
    def __init__(self, log):
        if log is None:
            self.log = logging.getLogger(type(self).__name__)
        else:
            self.log = log.getChild(type(self).__name__)
...
```

Then, when you instantiating the class on the CSC here

https://github.com/lsst-ts/ts_ataos/blob/cb64c48e203047915dda2c4a4be077ac022a62d5/python/lsst/ts/ataos/ataos_csc.py#L71

, you pass the CSC log attribute:

```
        self.model = Model(self.log)
```

Reply:
Thanks... that makes more sense...